### PR TITLE
Fix typo to init min_psync[] in unit test

### PR DESCRIPTION
**Pull request by naughtont3**
Signed-off-by: Thomas Naughton <naughtont@ornl.gov>

### CODE REVIEW

**Comment by lanl-ompi** [comment]
Can one of the admins verify this patch?

File: test/unit/reduce_active_set.c [hunk 1]
```diff
@@ -58,7 +58,7 @@ int main(void)
 
     for (i = 0; i < SHMEM_REDUCE_SYNC_SIZE; i++) {
         max_psync[i] = SHMEM_SYNC_VALUE;
-        max_psync[i] = SHMEM_SYNC_VALUE;
+        mix_psync[i] = SHMEM_SYNC_VALUE;
```
Comment (jdinan) [inline test/unit/reduce_active_set.c]: Whoops, I think you meant `min_psync`.

Comment (naughtont3) [reply test/unit/reduce_active_set.c]: Doh!  Yep, should be `min_psync`.  updated PR.

**Comment by jdinan** [comment]
Ready to merge once it clears CI testing.

**Pull request merged**